db/migrate/20150410124111_add_nic_provider_config_to_compute_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddNicProviderConfigToComputeSite < ActiveRecord::Migration
+  def change
+    add_column :atmosphere_compute_sites, :nic_provider_config, :text
+  end
+end
```

Comment:
I recommend to merge this migration with `AddNicProviderClassNameToComputeSite`. One without another IMHO does not have sense